README.md

```diff
@@ -6,7 +6,7 @@ Get started with the Microsoft Graph SDK for Go by integrating the [Microsoft Gr
 
 > **Note:** this SDK allows you to build applications using the [v1.0](https://docs.microsoft.com/graph/use-the-api#version) of Microsoft Graph. If you want to try the latest Microsoft Graph APIs under beta, use our [beta SDK](https://github.com/microsoftgraph/msgraph-beta-sdk-go) instead.
 >
-> **Note:** The Microsoft Graph Go SDK is currently in Release Candidate (RC) version starting from version 0.56.0. The SDK is still undergoing testing but minimum breaking changes should be expected. Checkout the [known limitations](https://github.com/microsoftgraph/msgraph-sdk-go-core/issues/1).
+> **Note:** The Microsoft Graph Go SDK is currently in General Availability version starting from version 0.60.0. The SDK is considered stable, regular releases and updates to the SDK will however continue weekly..
```

Comment:
we should bump version to 1.0.0 and make sure we bump the dependencies along the way

Reply:
v1.0 should be fine. I assume this means we wait and name this weeks release `v1.0.0` before merging this.

Reply:
correct